### DISCUSSION TITLE
What is your favorite cakes?

### DISCUSSION THREAD

**Discussion by MiyamotoAnju** [NAIST-SE/AttractingContributors#1156] [2024 Attraction Contributions]
https://github.com/MiyamotoAnju/AttractingContributors/tree/main

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite one.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by Ryota-keda** [comment]
I just made PR.
Could you contribute to my project?
https://github.com/Ryota-keda/Baseball

**Comment by ryosuke-sota** [comment]
I made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/ryosuke-sota/AttractingContributors.git 
Thank you.

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by Idescrew** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.

https://github.com/Idescrew/AttractingContributors

**Comment by H-S-0811** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.

https://github.com/H-S-0811/AttractingContributors
